spark-jobs/src/main/scala/za/co/absa/enceladus/spline/persistence/HadoopFsDataLineageWriter.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2017 ABSA Group Limited
```

Comment:
how does this pass the check? Don't we have a rule to say it should be 2018?

Also in JSONSerialization.scala
And has a range HadoopFsPersistenceFactory.scala

Reply:
Thanks for noticing, it must have been as I copied some of the files from spline prior to making changes to them. Fixed.

I think the licence check only checks if there is a "valid" licence, but for the ABSA one, it does not check the year.

Reply:
> Don't we have a rule to say it should be 2018?
@Zejnilovic what is that rule and why?